### PR TITLE
upgrade AngularJS to 1.3.3

### PR DESCRIPTION
**Pull request by kuisathaverat**
This upgrade the AngularLS to version 1.3.3 to avoid an XSS vulnerability 
https://github.com/angular/angular.js/issues/14939

@reviewbybees @oleg-nenashev

### CODE REVIEW

File: public/index.js [hunk 1]
```diff
@@ -90906,7 +90906,7 @@ define('kibana',['require','angular','lodash','jquery','modules','routes','elast
     // Use this for cache busting partials
     .constant('cacheBust', window.KIBANA_COMMIT_SHA)
     // The minimum Elasticsearch version required to run Kibana
-    .constant('minimumElasticsearchVersion', '1.4.4')
+    .constant('minimumElasticsearchVersion', '2.0.0')
```
Comment (oleg-nenashev) [inline public/index.js:90909]: Thanks for catching it!

**Comment by ghost** [comment]
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

**Comment by kuisathaverat** [comment]
@reviewbybees done

**Comment by oleg-nenashev** [comment]
Oh, it missed my review Queue. Anyway, I am fine with this change. I know how much time @kuisathaverat invested into testing

**Pull request merged**